docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -484,6 +484,17 @@ function sitenow_form_alter(&$form, FormStateInterface $form_state, $form_id) {
         }
       }
       break;
+
+    // Remove access to headline field in footer contact block.
+    case 'block_content_uiowa_text_area_edit_form':
+      $block = $form_state->getFormObject()->getEntity();
+      $uuid = $block->uuid();
+      // For Footer Contact Information, limit non-admins to minimal.
+      if ($uuid == '0c0c1f36-3804-48b0-b384-6284eed8c67e') {
+        /** @var Drupal\uiowa_core\Access\UiowaCoreAccess $check */
```

Comment:
This comment referencing the variable $check doesn't make sense here. It isn't doing a access check/using uiowa_core functionality.